### ISSUE TITLE
.env error

### ISSUE THREAD

**Issue by Rumyra** [jackfranklin/express-server-boilerplate#1]
This entirely could be *my* environment btw...

So this is awesome and I've used it as a starter a couple of times now. However both builds I get this error:

`Error: ENOENT: no such file or directory, open '.env'] errno: -2, code: 'ENOENT', syscall: 'open', path: '.env'`

...when starting in console (either by npm start or node server.js) - the app serves fine and if I touch .env the error goes away.

Apologies as this could just be my environment (it's been causing me other issues recently) and I haven't used dotenv b4 (although _that's_ cool) - I was gonna show u next Sat, but I just got the error again and wanted to make a note of it 🤓

**Comment by Rumyra** [comment]
OK yeh - I should have read the docs 😜 it seems it's just chucking it cos the files not there 😎

**Comment by jackfranklin** [comment]
Yeah it's because I rely on .env - just run `touch .env` and create an entirely blank one, error will go away.

I do this to create a new project from this boilerplate:

```
git clone --depth=1 --branch=master git@github.com:jackfranklin/express-server-boilerplate.git $1 && cd $1 && rm -rf .git && git init && git add -A && git commit -m "Initial commit" && touch .env && npm install
```
